### DISCUSSION TITLE
List events from tomorrow only

### DISCUSSION THREAD

**Discussion by modo1302** [totaldebug/atomic-calendar-revive#1084] [Support] [answered]
Hi there, is there any way to list events for next days and not show today's events?

**Comment by modo1302** [comment]
fyi managed to achieve my goal by hiding the entire 'todays' row in css (I know, sooo dirty!):
`tr:has(.event-leftCurrentDay){
              visibility: hidden;
            }`

so I'm using two atomic instances, one for todays events and another for the upcoming ones.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/133964285/4337eba5-0f97-4ff2-844e-647184df1e28)

If anyone knows a cleaner way to acheve this - please share!

**Comment by marksie1988** [comment, marked as answer]
You could use the option `startDaysAhead: 1`  this will show events 1 day ahead of the current date, which would show you the future events.
